pkgs/tools/networking/openvpn/default.nix

```diff
@@ -19,72 +19,58 @@
 
 let
   inherit (lib) versionOlder optional optionals optionalString;
+  withIpRoute = stdenv.isLinux && (versionOlder version "2.5.4");
```

Comment:
I suggest the following:

1. get rid of `withIpRoute` as we are no longer packaging 2.5
2. use the `finalAttrs:` pattern (for the lack of a better name)
3. move version and hash into the call to `fetchurl`.

However, I don't recall - why is iproute2 only needed for versions < 2.5.4?

Reply:
1. done
2. I could add it, but we are not currently using `rec`, so it wouldn't do anything. Do you mean something like using the `${finalAttrs.version}` in `src.url`? I was recommended against it in a [previous PR](https://github.com/NixOS/nixpkgs/pull/260887#discussion_r1359945399), fwiw.
3. That comes from 75741425ce52a783a65dbbf9633bcb9451ec5c46, apparently.